src/content/1.7/modules/sample_modules/order_view_page_new_hooks/additional-action-buttons.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Additional action buttons
+weight: 3
+---
+
+# Additional buttons in the main buttons bar
+
+## `actionGetAdminOrderButtons` hook demo
+
+We use this hook to display additional action buttons into the main buttons bar.
+
+Let's add hook related code to main module class `demovieworderhooks`:
+
+```php
+    /**
+     * Add buttons to main buttons bar
+     */
+    public function hookActionGetAdminOrderButtons(array $params)
+    {
+        $order = new Order($params['id_order']);
+
+        /** @var \Symfony\Bundle\FrameworkBundle\Routing\Router $router */
+        $router = $this->get('router');
+
+        /** @var \PrestaShopBundle\Controller\Admin\Sell\Order\ActionsBarButtonsCollection $bar */
+        $bar = $params['actions_bar_buttons_collection'];
+
+        $viewCustomerUrl = $router->generate('admin_customers_view', ['customerId'=> (int)$order->id_customer]);
+        $bar->add(
+            new \PrestaShopBundle\Controller\Admin\Sell\Order\ActionsBarButton(
+                'btn-secondary', ['href' => $viewCustomerUrl], 'View customer'
+            )
+        );
+        $bar->add(
+            new \PrestaShopBundle\Controller\Admin\Sell\Order\ActionsBarButton(
+                'btn-info', ['href' => 'https://www.prestashop.com/'], 'Go to prestashop'
+            )
+        );
+        $bar->add(
+            new \PrestaShopBundle\Controller\Admin\Sell\Order\ActionsBarButton(
+                'btn-dark', ['href' => 'https://github.com/PrestaShop/example-modules/tree/master/demovieworderhooks'], 'Go to GitHub'
+            )
+        );
+        $createAnOrderUrl = $router->generate('admin_orders_create');
+        $bar->add(
+            new \PrestaShopBundle\Controller\Admin\Sell\Order\ActionsBarButton(
+                'btn-link', ['href' => $createAnOrderUrl], 'Create an order'
+            )
+        );
+    }
+```
+
+{{% notice note %}}
+we used full path here `\PrestaShopBundle\Controller\Admin\Sell\Order\ActionsBarButton`
```

Comment:
Oh you added a note about it, great idea! It's a good compromise, just the capitalize missing

```suggestion
We used full path here `\PrestaShopBundle\Controller\Admin\Sell\Order\ActionsBarButton`
```